### ISSUE TITLE
build: run a subset of our roachtests on every PR (using --local)

### ISSUE THREAD

**Issue by danhhz** [cockroachdb/cockroach#23604]
This will both prevent some classes of regressions in roachtest itself as well as making sure basic integration tests work (running tpcc for a minute, etc)

Currently blocked on cockroachdb/roachprod#128

**Comment by jordanlewis** [comment]
Isn't this already done?

**Comment by danhhz** [comment]
For a while, it was only running on master builds (not PR builds) and it only had like one test whitelisted. @petermattis has porting acceptance tests lately, so maybe we can call it good now?
